en/docs/learn/api-controller/ci-cd-with-wso2-api-management.md

```diff
@@ -277,13 +277,15 @@ Run any of the following CTL commands to get keys for the API.
             
         -   Required :  
             `--environment` or `-e` : Key generation environment  
-            `--name` or `-n` : API to enerate keys for  
-            `--version` or `-v` : Version of the API  
-            `--provider` or `-r` : Provider of the API   
+            `--name` or `-n` : API or API Product to enerate keys for   
+        -   Optional :  
+            `--provider` or `-r` : Provider of the API or API Product  
+            `--version` or `-v` : Version of the API or API Product (Currently API Products do not have versions)
 
 !!! info
     - Upon running the above command, the CTL tool will create a default application in the environment, subscribe to the API, and generate keys based on the token type defined in the `<USER_HOME>/.wso2apictl/main-config.yaml`file. 
     - Using apictl tool the token type , HTTP request timeout, and export directory can be set up and changed. For more information on changing the token type, see [Set token type]({{base_path}}/learn/api-controller/getting-started-with-wso2-api-controller/#set-token-type), [Set HTTP request timeout]({{base_path}}/learn/api-controller/getting-started-with-wso2-api-controller/#set-http-request-timeout) and [Set export directory]({{base_path}}/learn/api-controller/getting-started-with-wso2-api-controller/##set-export-directory) accordingly. 
+    - When running the above command, if you have not specified the --version (-v), the tool will consider the version as 1.0.0 by default. If you have specified the version, then that value will be considered.
 
```

Comment:
Is this applicable for the APIs too? I know it is the behaviour for the API Products.

Reply:
Verified.